tests/e2e/vsphere_volume_expansion.go

```diff
@@ -2450,8 +2454,12 @@ func increaseOnlineVolumeMultipleTimes(ctx context.Context, f *framework.Framewo
 	expectEqual(len(pvcConditions), 0, "pvc should not have conditions")
 
 	ginkgo.By("Verify filesystem size for mount point /mnt/volume1")
-	fsSize, err = getFSSizeMb(f, pod)
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	if windowsEnv{
+		fsSize = getWindowsPodSize(client, pod)
+	}else{
+		fsSize, err = getFSSizeMb(f, pod)
```

Comment:
is `getFSSizeMb()` returning correct value for windows pods?

Reply:
no, we have `getWindowsDiskSize` method to get correct value for windows pods.